docs/platform/block-storage/block-storage-use-cases/index.md

```diff
@@ -8,58 +8,80 @@ license: '[CC BY-ND 4.0](https://creativecommons.org/licenses/by-nd/4.0)'
+While the Block Storage service has full support for hot swapping, it is important to follow the detachment instructions outlined in our [Using Block Storage](/docs/platform/block-storage/how-to-use-block-storage-with-your-linode/#how-to-detach-a-block-storage-volume-from-a-linode) guide. If a Volume is not safely detached, there is a risk of data loss for the Volume.
 {{< /caution >}}
 
 ### Container Storage
 
-Containers, like those created with Docker or Kubernetes Pods, can benefit from having some type of persistent storage. This helps to keep the container size down, and makes it easy to maintain data despite the lifecycle of the container or Pod. If you are using Docker, you can use the [Docker Volume Driver for Linode](https://github.com/linode/docker-volume-linode) to create a Docker volume from a Block Storage Volume. Similarly, if you are using Kubernetes you can use the [Container Storage Interface (CSI) Driver for Linode Block Storage](https://github.com/linode/linode-blockstorage-csi-driver) to create a Block Storage Volume backed Persistent Volume Claim.
+Containers, like those created with Docker or inside Kubernetes Pods, can benefit from having some type of persistent storage. This helps to keep a container's size down and makes it easy to maintain data outside of the normal lifecycle of the container or Pod.
+
+If you are using Docker, you can use the [Docker Volume Driver for Linode](https://github.com/linode/docker-volume-linode) to create a Docker volume from a Block Storage Volume. Similarly, if you are using Kubernetes you can use the [Container Storage Interface (CSI) Driver for Linode Block Storage](https://github.com/linode/linode-blockstorage-csi-driver) to create a Persistent Volume Claim that's backed by a Block Storage Volume.
 
 ### Running Cloud Software
 
-In a climate where ownership over one's own data is an important need for many individuals and organizations, hosting your own cloud software is a great use case for Object Storage. Simply create and mount a Block Storage Volume, [install an application like OwnCloud](https://linode.com/docs/applications/cloud-storage/install-and-configure-owncloud-on-ubuntu-16-04/), and point the data folder to a location on your Volume. If you ever run out of space on your Volume you can always increase its size.
+In a climate where ownership over one's data is an important need for many individuals and organizations, hosting your own cloud software is a great use case for Block Storage. Create and mount a Block Storage Volume, [install an application like OwnCloud](/docs/applications/cloud-storage/install-and-configure-owncloud-on-ubuntu-16-04/), and point its data folder to a location on your Volume. If you ever run out of space on your Volume you can always increase its size.
```

Comment:
Removed "Simply" to get away from adjectives that may or may not describe how easy a given user would perceive a task